dev-haskell/libbf/libbf-0.6.8.ebuild

```
@@ -0,0 +1,35 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+#hackport: flags: -system-libbf
+
+# ebuild generated by hackport 0.9.0.0.9999
+
+CABAL_PN="libBF"
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="A binding to the libBF library"
+HOMEPAGE="https://hackage.haskell.org/package/libBF"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+
+RDEPEND=">=dev-haskell/hashable-1.3:=[profile?]
+	>=dev-lang/ghc-9.0.2:=
+"
+
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-3.4.1.0
+	test? ( >=dev-haskell/tasty-1.3 <dev-haskell/tasty-1.6
+		>=dev-haskell/tasty-hunit-0.10 <dev-haskell/tasty-hunit-0.11 )
+"
+
+/src_configure() {
```

```suggestion
src_configure() {
```

Not sure how this wasn't caught by `pkgcheck`?

---

Good pickup!